WS2012R2/lisa/remote-scripts/ica/Check_FCOPY_Daemon.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
```

Comment:
Pls rename this file to `FCOPY_Check_Daemon.sh`. Watch out to change the name also in other scripts.